mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -34,6 +34,10 @@ CLASS net/minecraft/class_37 net/minecraft/entity/Entity
 	FIELD field_3567 prevPitch F
 	FIELD field_3568 boundingBox Lnet/minecraft/class_39;
 	FIELD field_3569 onGround Z
+	METHOD <init> (Lnet/minecraft/class_26;)V
+		ARG 1 world
+	METHOD equals (Ljava/lang/Object;)Z
```

Comment:
Since equals doesn't necessarily have to take an Entity, as it is Object, I would leave this unmapped.